### ISSUE TITLE
npm run prepublish fails

### ISSUE THREAD

**Issue by lgandecki** [xolvio/chimp#411]
#####  Expected behaviour
```
npm run prepublish

> chimp@0.35.0 prepublish /Users/lukaszgandecki/code/chimp
> node ./scripts/prepublish.js

> Transpiling ES2015
> Completed transpiling ES2015


```
#####  Actual behaviour

```
npm run prepublish

> chimp@0.35.0 prepublish /Users/lukaszgandecki/code/chimp
> node ./scripts/prepublish.js

> Transpiling ES2015
SyntaxError: src/bin/chimp.js: 'return' outside of function (57:2)
  55 | if (argv.v || argv.version) {
  56 |   console.log(pkg.version);
> 57 |   return;
     |   ^
  58 | }
  59 |
  60 | if (argv.host && ((argv.host.indexOf('sauce') !== -1 || argv.host.indexOf('browserstack') !== -1) || argv.host.indexOf('testingbot') !== -1)) {
> Completed transpiling ES2015

```
##### Steps to reproduce the problem

checkout master and do 
npm run prepublish

#####  Used version

* Chimp: 0.35.0
* Node.js: 4.4.0
* NPM: 2.14.20
* Operation system:  MacOS 10.11.1

#####  Console / Log Output

As above
------- 
*Join our Slack [xolv.io/community](http://xolv.io/community) #chimp channel, where you can find help and help others.*

**Comment by samhatoum** [comment]
I fixed this on the plane, let me check it in!

**Comment by lgandecki** [comment]
Quick fix, I've got it to work too, wanted to add an issue first and link
Pr to it.
but I'm curious to see your solution :)

Sent from my iPhone

On Jul 6, 2016, at 19:54, Sam Hatoum <notifications@github.com> wrote:

I fixed this on the plane, let me check it in!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/xolvio/chimp/issues/411#issuecomment-230852335>, or mute
the thread
<https://github.com/notifications/unsubscribe/AD0S7zvDhdlsn4f_q7eFP035NL0Ybem6ks5qS-u5gaJpZM4JGVjG>
.

**Comment by samhatoum** [comment]
```
  if (argv.v || argv.version) {
    console.log(pkg.version);
  } else if (!argv.port) {
  // ...
```

**Comment by lgandecki** [comment]
well, same thinking ;)

**Comment by samhatoum** [comment]
Fixes in 0.34.1
